backend/src/main/java/io/papermc/hangar/db/dao/internal/projects/HangarProjectPagesDAO.java

```diff
@@ -21,15 +21,14 @@ public interface HangarProjectPagesDAO {
         "   exists(SELECT 1 FROM project_home_pages php WHERE php.page_id = pp.id AND php.project_id = p.id) AS home" +
         "   FROM project_pages pp" +
         "       JOIN projects p ON pp.project_id = p.id" +
```

Comment:
I think this joins can be removed, right? 
the inner query above can just use pp.project_id, same in the where